product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/65b_import_foreign_schema.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "IMPORT FOREIGN SCHEMA"
+---
+
+## Name
+
+`IMPORT FOREIGN SCHEMA` &mdash; Import a foreign schema from a foreign server or database link.
```

Comment:
Do you think it should be "Import a foreign schema from a foreign server using a database link" I think we can check on this with dev or QA what is right technically.

Reply:
@surajkharage19  you mentioned that to perform an import you either need a foreign server or a database link. However, as Nidhi asked here, I thought you needed a database link first, to then use the link to perform the import? 

Can you provide some clarification, please?

Reply:
There are 2 ways to access the remote database. One of them is using FDWs and other one is using database link. 
In FDW approach, we create server, user mapping and foreign table. In database link approach, we create database link using CREATE DATABASE LINK command and this will internally create server and user mapping.

Import foreign schema feature belongs to FDW specifically but this also works with the database links. As mentioned above, for database link internally, it creates server and user mapping, so using that server, we can import foreign schema as well. So current statement looks correct to me.